src/openms/include/OpenMS/FILTERING/TRANSFORMERS/SpectraMerger.h

```diff
@@ -566,31 +657,33 @@ namespace OpenMS
           Size spec_a = consensus_spec.size(), spec_b = exp[*sit].size(), align_size = alignment.size();
           for (auto pit = exp[*sit].begin(); pit != exp[*sit].end(); ++pit)
           {
-            if (alignment.empty() || alignment[align_index].second != spec_b_index)
+            if (alignment.size() == 0 || alignment[align_index].second != spec_b_index)
```

Comment:
I think @timosachsenberg will not like this :)

Reply:
Haha. Clang-tidy complains. But yeah instead of size() > 0 it is considered slightly better to write !empty()

Reply:
Wow Is it? Thanks all. I fix them.